### PR TITLE
Code formatting and code improvement

### PR DESCRIPTION
**Pull request by sudoii**

### CODE REVIEW

**Review by eliottvincent — APPROVED** [review]

**Review by eliottvincent — CHANGES_REQUESTED** [review]
We'll keep our 2-spaces indentation 🙂

**Review by eliottvincent — APPROVED** [review]

**Review by eliottvincent — CHANGES_REQUESTED** [review]
The string formatting looks fine, but we'll keep the 2-spaces indentation.

File: crisp_api/__init__.py [hunk 1]
```diff
@@ -101,16 +101,16 @@ def __build_head_result(self, response):
...
+            result["reason"] = "found"
+        elif response.status_code == 401:
+            result["reason"] = "unauthorized"
+        elif response.status_code == 403:
+            result["reason"] = "not_allowed"
+        elif response.status_code == 404:
+            result["reason"] = "not_found"
         else:
-            result["reason"] = "unsuported_code";
+            result["reason"] = "unsuported_code"
```
Comment (eliottvincent) [inline crisp_api/__init__.py]: Ok to trim the `;`, however we prefer keeping the parentheses.

File: crisp_api/__init__.py [hunk 2]
```diff
@@ -84,7 +84,7 @@ def __do_request(self, method, resource, query=None, data=None):
             headers=self.REQUEST_HEADERS,
             auth=auth,
             params=query,
-            data=(json.dumps(data) if data != None else None)
+            data=(json.dumps(data) if data is not None else None)
```
Comment (eliottvincent) [inline crisp_api/__init__.py:87]: This is right, thanks!

**Comment by eliottvincent** [comment]
Hey @sudoii , thanks a lot for this PR! I've reviewed it, let me know when requested changes are done 🙂

File: crisp_api/__init__.py (hunk 2 above)
Comment (sudoii) [reply crisp_api/__init__.py:87]: I will submit a commit to change to 2-spaces indentation and restore "()"

**Comment by sudoii** [comment]
> Hey @sudoii , thanks a lot for this PR! I've reviewed it, let me know when requested changes are done 🙂

heyyy @eliottvincent , I have changed to 2-spaces indentation and kept "()"

**Comment by eliottvincent** [comment]
https://pypi.org/project/crisp-api/1.1.4/ 😉

**Pull request merged**